src/Main.elm

```diff
@@ -156,28 +144,23 @@ view siteMetadata page =
     StaticHttp.succeed
         { view =
             \model viewForPage ->
-                Layout.view (pageView model siteMetadata page viewForPage) page
+                Layout.view (pageView siteMetadata page viewForPage) page
         , head = head page.frontmatter
         }
 
 
 pageView :
-    Model
-    -> List ( PagePath Pages.PathKey, Metadata )
+    List ( PagePath Pages.PathKey, Metadata )
     -> { path : PagePath Pages.PathKey, frontmatter : Metadata }
     -> Rendered
     -> { title : String, body : List (Element Msg) }
-pageView model siteMetadata page viewForPage =
+pageView siteMetadata page viewForPage =
```

Comment:
Similar thinking here to my comment about the `subscriptions`. `model` is not used, but it's nice to know that it's available when you clone the starter repo.

By the way, this design is going to change quite a bit with this template-module design I'm working on: https://github.com/dillonkearns/elm-pages/pull/109. But for now, I think leaving the argument to help people understand what they have access to is good for educational purposes.

Reply:
Sure thing.